conbench/entities/benchmark_result.py

```diff
@@ -236,24 +243,65 @@ class BenchmarkResultSerializer:
 
 
 class _BenchmarkFacadeSchemaCreate(marshmallow.Schema):
-    run_id = marshmallow.fields.String(required=True)
-    run_name = marshmallow.fields.String(required=False)
-    run_reason = marshmallow.fields.String(required=False)
+    run_id = marshmallow.fields.String(
+        required=True,
+        metadata={"description": "Unique identifier for a run of benchmarks."},
+    )
+    run_name = marshmallow.fields.String(
+        required=False,
+        metadata={
+            "description": "Name for the run. When run in CI, this should be of the style '{run reason}: {commit sha}'."
+        },
+    )
+    run_reason = marshmallow.fields.String(
+        required=False,
+        metadata={
+            "description": "Reason for run (commit, pull request, manual, etc). This should be low cardinality. 'commit' is a special run_reason for commits on the default branch which are used for history"
+        },
+    )
     batch_id = marshmallow.fields.String(required=True)
-    timestamp = marshmallow.fields.DateTime(required=True)
+    timestamp = marshmallow.fields.DateTime(
+        required=True, metadata={"description": "Timestamp the benchmark ran"}
```

Comment:
started? completed? Does it matter?

Reply:
I had the exact same debate, IMO it probably doesn't matter and can be left ambiguous or up to specific implementations to care about. But at least let's leave the docs ambiguous for now

Reply:
We removed some of that ambiguity. Related:
https://github.com/conbench/conbench/pull/651#discussion_r1092925503
https://github.com/conbench/conbench/blob/fbf5a6ce898532d44327e3ac961f5abb852843bc/conbench/entities/benchmark_result.py#L507